src/NServiceBus.Core/obsoletes.cs

```diff
@@ -1819,4 +1820,35 @@ namespace NServiceBus.Transports.Msmq.Config
     public class MsmqSettings
     {
     }
-}
\ No newline at end of file
+}
+
+public static class ConfigureHandlerSettings
+{
+    [ObsoleteEx(
+         RemoveInVersion = "7.0",
+         TreatAsErrorFromVersion = "6.0",
+         Message = "Setting property values explicitly is no longer supported via the builtin container in NServiceBus. Property injection is still enabled by default. Use the `.ConfigureComponent(b=> new MyMessageHandler(){ MyProperty = X})` if you want full controll over property values. Other option is to use one of the supported containers and configure property injection using their native configuration API's.")]
```

Comment:
I reworded the obsolete message to be more inline with the doco pull. 

> Setting property values explicitly is no longer supported via this API. Use `.ConfigureComponent(b=> new MyMessageHandler(){ MyProperty = X})` to get full control over handler creation.

Is that better?

Or perhaps just

> Use `.ConfigureComponent(b=> new MyMessageHandler(){ MyProperty = X})` to get full control over handler creation.

Thoughts?